From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 12:00:19 +0200
Subject: fix_kcm_category
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

The upstream kcm_cron.desktop file specifies "system-administration" as the
category for the kde control module, but this category was removed in KF5.
As a result, the control module is not visible in systemsettings.

Author: Jeremy Lainé <jeremy.laine@m4x.org>
Bug: https://bugs.kde.org/show_bug.cgi?id=347028
---
 src/kcm_cron.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/kcm_cron.desktop b/src/kcm_cron.desktop
index c338267..4c54c17 100644
--- a/src/kcm_cron.desktop
+++ b/src/kcm_cron.desktop
@@ -9,7 +9,7 @@ X-KDE-ServiceTypes=KCModule
 X-KDE-Library=kcm_cron
 X-KDE-ParentApp=kcontrol
 
-X-KDE-System-Settings-Parent-Category=system-administration
+X-KDE-System-Settings-Parent-Category=session
 
 Name=Task Scheduler
 Name[ar]=مجدول المهامّ
